src/utils/post_utils.js

```diff
@@ -72,7 +72,9 @@ export function canEditPost(state, config, license, teamId, channelId, userId, p
         if (hasNewPermissions(state)) {
             canEdit = canEdit && haveIChannelPermission(state, {team: teamId, channel: channelId, permission: Permissions.EDIT_POST});
             if (!isOwner) {
-                canEdit = canEdit && haveIChannelPermission(state, {team: teamId, channel: channelId, permission: Permissions.EDIT_OTHERS_POSTS});
+                // I only can edit my posts, at least until phase-2
```

Comment:
I could sworn I had posted this comment already, but it vanished...

Is the only thing stopping this from working a client-side check? Do we need server changes to complement this change?

Reply:
Same here, Server change not needed because it is intentionally disabled only in the interface.